src/util/Injected.js

```diff
@@ -207,6 +215,15 @@ exports.LoadUtils = () => {
                     preview.subtype = 'url';
                     options = { ...options, ...preview };
                 }
+            } else {
+                const link = window.Store.Validators.findLink(content);
+                if (link) {
+                    const preview = await window.Store.getLinkPreview(link).catch(() => null);
+                    if (preview) {
+                        preview.data.subtype = 'url';
+                        options = { ...options, ...preview.data };
+                    }
```

Comment:
I think you can't use `.catch()` with `await`
either `async`/`await` with `try...catch` block or `.then()` `.catch()`
```js
window.Store.getLinkPreview(link)
    .then(preview => {
        if (preview) {
            preview.data.subtype = 'url';
            options = { ...options, ...preview.data };
        }
    })
    .catch(() => null);
```

Reply:
Please use await with try and catch

Reply:
```js
try {
    const preview = await window.Store.getLinkPreview(link)
    if (preview) {
        preview.data.subtype = 'url';
        options = { ...options, ...preview.data };
    }
} catch (error) { /** empty */ }
```